elevate-cpanel

```diff
@@ -9885,7 +9887,7 @@ CloudLinux 7 => CloudLinux 8
        --log                                     Show the current elevation log.
        --status                                  Check the current elevation status.
        --clean                                   Cleanup scripts and files created by elevate-cpanel.
-       --leappbeta                               Use the special beta repo from leapp. Do this only if instructed to by cPanel.
+       --leappbeta                               Use the special beta repo from leapp (RHEL® derivatives only). Do this only if instructed to by cPanel.
```

Comment:
does this fail if on ubuntu?

Reply:
Yes, it will die early with a message saying that `BETA is unavailable for this leapp repo.`